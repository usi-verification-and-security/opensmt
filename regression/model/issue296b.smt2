(set-logic QF_LRA)
(declare-fun a () Real)
(assert (= a (/ 1 2)))
(check-sat)
(get-value (a))
(exit)
