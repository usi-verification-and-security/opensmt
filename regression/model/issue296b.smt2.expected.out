sat
((a (/ 1 2)))
